app.js

```diff
@@ -31,4 +31,62 @@ new SurveyPicture('tauntaun', 'img/tauntaun.jpg');
 new SurveyPicture('unicorn', 'img/unicorn.jpg');
 new SurveyPicture('usb', 'img/usb.jpg');
 new SurveyPicture('water-can', 'img/water-can.jpg');
-new SurveyPicture('wine-glass', 'img/wine-glass.jpg');
\ No newline at end of file
+new SurveyPicture('wine-glass', 'img/wine-glass.jpg');
+
+// puts pictures in the form
+
+SurveyPicture.displayPictures = function() {
+  var pics = document.getElementById('pictures');
+  for (var i = 0 ; i < allPictures.length; i++) {
+    var namesForPictures = document.createElement('option');
+    namesForPictures.textContent = allPictures[i].name;
+    pics.appendChild(namesForPictures);
+  }
+};
+SurveyPicture.displayPictures();
+
+// event listener
+
+submitForm.addEventListener('submit', submitingPictures);
+
+// turns ray into string
+
+function savingCart() {
+  localStorage.filepath = JSON.stringify(itemsSelected);
+}
+
+function submitingPictures (event) {
+  event.preventDefault();
+  console.log(event.target.pictures.value);
+  for (var i = 0; i < allPictures.length; i++) {
+    if (event.target.pictures.value === allPictures[i].name) {
+      itemsSelected[i]= allPictures[i].filepath;
+    }
+  }
+  savingCart();
+  event.target.reset();
+}
+
+var allPictures = [];
```

Comment:
You are resetting `allPictures` to an empty array after you have filled it above. This is breaking quite a bit of your functionality.